src/storage/mvcc/txn.rs

```diff
@@ -199,14 +178,40 @@ impl<S: Snapshot> MvccTxn<S> {
         Ok(())
     }
 
+    fn handle_non_pessimistic_lock_conflict(
+        &self,
```

Comment:
Seems this function never use self.

Reply:
It is a bit complex to call it without self.